sql/sql_admin.cc

```diff
@@ -1760,7 +1760,7 @@ bool Sql_cmd_analyze_table::execute(THD *thd) {
   });
 
   if (WSREP(thd) && !thd->lex->no_write_to_binlog &&
-      wsrep_to_isolation_begin(thd, NULL, NULL, first_table)) {
+      wsrep_to_isolation_begin(thd, first_table->db, NULL, first_table)) {
```

Comment:
why this change?

Reply:
As said in my previous comment, we need to know the db name of the table which are running ANALYZE TABLE on. So, it is better to send the db name if we already have that information.